android-sdk/android/src/main/kotlin/io/logto/sdk/android/auth/webview/WebViewAuthActivity.kt

```diff
@@ -0,0 +1,56 @@
+package io.logto.sdk.android.auth.webview
+
+import android.annotation.SuppressLint
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import android.webkit.WebSettings
+import android.webkit.WebView
+import androidx.appcompat.app.AppCompatActivity
+import io.logto.sdk.android.auth.LogtoAuthManager
+
+class WebViewAuthActivity : AppCompatActivity() {
+    private lateinit var webView: WebView
+
+    @SuppressLint("SetJavaScriptEnabled")
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        supportActionBar?.hide()
+
+        val uri = intent.getStringExtra(EXTRA_URI)
+
+        if (uri !== null) {
+            webView = WebView(this).apply {
+                settings.javaScriptEnabled = true
+                settings.cacheMode = WebSettings.LOAD_NO_CACHE
+                webViewClient = WebViewAuthClient(this@WebViewAuthActivity)
+            }
+
+            WebViewSocialHandler.injectToWebView(webView, this)
```

Comment:
the mixing of single / multi instance is confusing. shall we just follow the same pattern unless we meet blockers?

Reply:
updated